modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/DownloadHelper.js

```diff
@@ -20,7 +20,7 @@ const getEndpoint = (type, externalReferenceCode) => {
 	const endpoints = {
 		batchPlannerTemplate: `/o/batch-planner/v1.0/plans/${externalReferenceCode}/template`,
 		errorReport: `${HEADLESS_BATCH_ENGINE_URL}/import-task/by-external-reference-code/${externalReferenceCode}/failed-items/report`,
-		importFile: `${HEADLESS_BATCH_ENGINE_URL}/import-task/by-external-reference-code/${externalReferenceCode}/content`,
```

Comment:
No, it is intended for the case of downloading the original import file.
Please implement for case export separately.

Reply:
Ok, my mistake, I implemented separate export and import cases.